### DISCUSSION TITLE
Recorder Mode sbase

### DISCUSSION THREAD

**Discussion by SirFunneh** [seleniumbase/SeleniumBase#2646] [Q&A] [answered]
Hi Michael, I am trying out the recorder mode function on my windows computer. Not very sure what I require to download or how to I go about starting it. Would I be able to start recorder mode from Visual Studio Code?

![sbase recorder](https://github.com/seleniumbase/SeleniumBase/assets/98967152/47d56759-c61f-4169-a2cb-19b6bb8cc04b)

**Comment by mdmintz** [comment, marked as answer]
If `seleniumbase` is installed, then you'll have access to `sbase` [console commands](https://github.com/seleniumbase/SeleniumBase/blob/master/seleniumbase/console_scripts/ReadMe.md):

```bash
pip install seleniumbase
```
